Description: Make systemd units friendly to packaging
Author: Jonathon Fernyhough <jonathon.fernyhough@york.ac.uk>

--- i/etc/systemd/system/ballast.service
+++ w/etc/systemd/system/ballast.service
@@ -5,7 +5,7 @@
 ConditionPathExists=/etc/ballastrc

 [Service]
-ExecStart=/usr/local/sbin/ballastd
+ExecStart=/usr/sbin/ballastd
 Type=forking
 PIDFile=/var/run/ballastd.pid
 ExecReload=/bin/kill -HUP $MAINPID
--- i/etc/systemd/system/ballast-agent.service
+++ w/etc/systemd/system/ballast-agent.service
@@ -6,4 +6,4 @@

 [Service]
 Type=oneshot
-ExecStart=/usr/local/sbin/ballast-agent
+ExecStart=/usr/sbin/ballast-agent
